src/main/java/AddTaskCommand.java

```diff
@@ -0,0 +1,50 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+/**
+ * Generates the correct type of task and adds it to the task list
+ */
+
+public class AddTaskCommand extends Command {
+
+    private String input; // Details of the task
+    private String type; // Type of task
+    private boolean isComplete = false;
+
+    public AddTaskCommand(String input, String type) {
+        this.input = input;
+        this.type = type;
+    }
+
+    public AddTaskCommand(String input, String type, boolean isComplete) {
+        this.input = input;
+        this.type = type;
+        this.isComplete = isComplete;
+    }
```

Comment:
I like your use of inheritance when structuring the Command classes as it makes for clean and logical code.